code/modules/clothing/masks/boxing.dm

```diff
@@ -18,7 +18,7 @@
 	desc = "Worn by robust fighters, flying high to defeat their foes!"
 	icon_state = "luchag"
 	item_state = "luchag"
-	flags = BLOCKHAIR
+	flags = BLOCKHAIR | MASKCOVERSMOUTH
 	flags_inv = HIDEFACE
 	w_class = 2

```

Comment:
![red-front-big_1](https://cloud.githubusercontent.com/assets/7305553/7666922/3459b5f4-fbef-11e4-9bf3-310dd82edba7.jpg)
If you can't eat through that, you aren't hungry enough.